Remotion/Mixins/UnitTests/Core/CodeGeneration/SetUpFixture.cs

```diff
@@ -71,6 +71,7 @@ public static void AddSavedAssembly (string assemblyPath)
     [OneTimeSetUp]
     public void OneTimeSetUp ()
     {
+      Environment.CurrentDirectory = TestContext.CurrentContext.TestDirectory;

```

Comment:
we have been able to work (almost) compeltely without setting Env.CurDir when @dkudernatsch did the other conversions. we should try to keep it this way.

See https://re-motion.atlassian.net/browse/RM-7445

Please check if it is really not sensibly possible to use absolute paths in this instance. If not, please add to the aformentioned jira ticket.

Reply:
I'll take another look at this next week, but some `Assembly.Save` method (if I remember correctly) only accepts filenames and no paths, which makes it a little harder

Reply:
I thought about the same thing while reviewing, but could not find a nice way to work around setting `Environment.CurrentDirectory`.

Reply:
Should be timeboxed.

Reply:
okay, I think this might be a one-liner. Didn't try it yet, but I know what to look for in the next review :)